scrapers/JulesJordan.yml

```diff
@@ -29,10 +43,54 @@ xPathScrapers:
         Name: $details//span[@class="update_dvds"]/a/text()
       Image:
         selector: //script[contains(text(),'useimage')]/text()
-        replace:
-          - regex: (?:.+useimage = "([^"]+).+)
-            with: $1
-          - regex: (^\/)
-            with: https://www.julesjordan.com/
+        postProcess:
+          - replace:
+            - regex: (?:.+useimage = "([^"]+).+)
+              with: $1
+            - regex: (^\/)
+              with: https://www.julesjordan.com/
+
+  performerScraper:
+    performer:
+      Name: //span[@class='title_bar_hilite']/text()
+      Birthdate:
+        selector: //div[@class='model_bio model_bio_info']//text()[contains(.,'Age')]
+        postProcess:
+          - replace:
+            - regex: "Age:"
```

Comment:
In some performers birthdate is missing and only the age eg `18` is present.
Not really much can be done about it though and constructing a regex to only match birth dates is probably a waste of time.